server/db.js

```diff
@@ -1,23 +1,26 @@
 const { createPool } = require('mysql2');
 const path = require('path');
+
 const {
   CREATE_CHAT_TABLE_QUERY,
   CREATE_PRODUCT_TABLE_QUERY,
   CREATE_USER_TABLE_QUERY,
+  CREATE_CHATROOM_TABLE_QUERY,
 } = require('./query/createTableQuery');
 
 require('dotenv').config({ path: path.join(__dirname, '/config/.env') });
 
 const createdPool = createPool({
-  host: process.env.DB_HOST,
-  user: process.env.DB_USER,
-  database: process.env.DB_DATABASE,
+  host: process.env.DB_HOST || 'localhost',
+  user: process.env.DB_USER || 'root',
+  database: process.env.DB_DATABASE || 'deal',
+  password: process.env.DB_PASSWORD ,
```

Comment:
DB 정보가 노출되지 않게 .env로 감싼 것인데 기본값을 넣어주면 의도에 위배되는 것 같습니다...!
```suggestion
  host: process.env.DB_HOST || 'localhost',
  user: process.env.DB_USER || 'root',
  database: process.env.DB_DATABASE || 'deal',
  password: process.env.DB_PASSWORD ,
```

Reply:
수정하도록 하겠습니다!!